src/api/specimen/GetSpecimenAnnotations.ts

```diff
@@ -0,0 +1,37 @@
+/* Import Dependencies */
+import axios from 'axios';
+
+/* Import Types */
+import { SpecimenAnnotations, Annotation } from 'global/Types';
+
+
+const GetSpecimenAnnotations = async (handle: string) => {
+    if (handle) {
+        let specimenAnnotations = <SpecimenAnnotations>{};
+
+        const endPoint = `specimens/${handle}/annotations`;
+
+        await axios({
+            method: "get",
+            url: endPoint,
+            responseType: 'json',
+        }).then((result) => {
+            const annotations: Annotation[] = result.data;
+
+            /* Refactor Annotations object */
```

Comment:
Is this still correct? Add a backlog item for this

Reply:
I think this is still preferred, by splitting it per property I can easily access the annotations of for example: the scientific name, in the view. That is why I created a separate function for this, only for the Specimen page since the other stuff just lists all the annotations.